tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -22,20 +26,23 @@ def test_bootstrap_numeric_metric_range_multi_batch_parameter_builder_bobby(
         bobby_columnar_table_multi_batch_deterministic_data_context
     )
 
-    # BatchRequest yielding two batches (January, 2019 and February, 2019 trip data)
+    # BatchRequest yielding three batches
     batch_request: dict = {
         "datasource_name": "taxi_pandas",
         "data_connector_name": "monthly",
         "data_asset_name": "my_reports",
     }
 
-    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(
-        name="row_count_range",
-        metric_name="table.row_count",
-        estimator="bootstrap",
-        false_positive_rate=1.0e-2,
-        round_decimals=0,
-        data_context=data_context,
+    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder
+    numeric_metric_range_parameter_builder = (
```

Comment:
@NathanFarmer Can we please join these lines -- there appears to be only one `numeric_metric_range_parameter_builder` in this test method.  Thanks!

Reply:
@alexsherstinsky it was complaining that the line was too long. Which do we care more about stylistically?

Reply:
@alexsherstinsky I have reverted the change